templates/repo/branch/list.tmpl

```diff
@@ -75,7 +75,11 @@
 									</td>
 									<td class="two wide right aligned">
 										{{if not .LatestPullRequest}}
-											{{if and (not .IsDeleted) $.AllowsPulls (gt .CommitsAhead 0)}}
+											{{if .IsIncluded}}
+												<div class="ui poping up orange small label" data-content="{{$.i18n.Tr "repo.branch.included_desc"}}" data-variation="tiny inverted" data-position="top right">
```

Comment:
Styling is not my strongest, but why `<div>` here and `<a>` below? One is inline and the other is a block element.

Reply:
was becose of copy from old PR